NEWS.md

```diff
@@ -1,73 +1,62 @@
-3.22.1
+3.23.1
 ======
 
-## Highlights
 
- * Sending SNMP traps: Using the new `snmp()` destination, incoming log messages
-   can be converted to SNMP traps, as the fields of the SNMP messages can be
-   customized with macros. (#2693)
+## Features
 
- * `$(template)` dynamic binding: Extends the $(template) template function to
-   allow dynamic binding. For example, the name of the template to be invoked
-   can come from the message (name-value pairs). (#2716)
+ * The `redis()` destination now handles any number of command parameters.
+   (#2816)
 
- * `syslog()`, `network()`: Add `dynamic-window-size()` option to enable dynamic
-   flow control that distributes the specified amount of window between active
-   connections at runtime. This can be used in low-memory environments, where
-   only a small subset of the active clients sends messages at high rate.
-   (#2772)
+ * The `format()` option of `date-parser()` supports lists. From now on, a
+   single `date-parser()` instance is able to process different date formats,
+   making it easy to catch on when some programs change the way they log
+   information. (#2779)
 
-## Features
+ * Add relocation support for `disk-buffer()`. The `relocate` subcommand of
+   `dqtool` can be used to move a single or multiple queue files. (#2855)
 
- * `match()`: Add support for the `template()` option (#2715)
- * `add-contextual-data()`: Allow using templates in name-value pairs (#2711)
- * Add support for floating point operations in template functions (#2742)
- * Add support for usec precision when parsing time (#2709)
+ * `file(), pipe()`: The `time-reap()` option now can be set or disabled for
+   each destination separately.
+   Use `time-reap(0)` to disable closing idle destination files. (#2798)
+
+ * `syslog-ng-ctl`: Add `list-files` subcommand to print files present in the
+   current configuration. (#2797)
 
 ## Bugfixes
 
- * Fix null pointer access when destinations are suspended (#2778)
- * Fix `grouping-by()` deadlock (#2758)
- * Fix a general source-related crash and enhance `wildcard-file()`'s bookmark
-   handling (#2589)
- * Fix infinite loop (reload/reopen) (#2739)
- * Fix `python()` package/module name collision (#2438)
- * Fix escaped quote in block argument (#2781)
- * Reintroduce test on SYSLOG_NG_HAVE_TIMEZONE (#2774)
- * `snmp()`: Fix template leak (#2746)
+ * Fix minor memory leaks (#2868)
+ * Add global context to standalone parsers (#2876)
+ * Fix heap usage after free in cfg_run_parser_with_main_context (#2884)
+ * Fix g_thread_init call order issue with older glibs (#2853)
+ * SNMP destination: fixing statistics format (#2854)
+ * eventlog: flush escaped_buffer when full (#2837)
+ * dbparser: remove unnecessary lock (#2838)
+ * dbparser: fix crash when context times out in the middle of another rule (#2832)
+ * radix: fix grouping in PCRE (#2808)
+ * add-contextual-data: make filters config plugin aware (#2886)
+ * Fix distro packages build (#2883)
```

Comment:
IMHO no need to announce this. Fixed a regression.

Reply:
Indeed, thanks!